Upstream: yes
Src: https://github.com/SFML/SFML/commit/119195ac20866fcecc93e33cda457b3fe54b1b0c.patch
Reason: error: invalid cast from type 'std::nullptr_t' to type 'void*

From 119195ac20866fcecc93e33cda457b3fe54b1b0c Mon Sep 17 00:00:00 2001
From: Dmitry Marakasov <amdmi3@amdmi3.ru>
Date: Tue, 16 Oct 2018 18:08:13 +0300
Subject: [PATCH] Remove unneeded cast

---
 src/SFML/Window/Unix/WindowImplX11.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/SFML/Window/Unix/WindowImplX11.cpp b/src/SFML/Window/Unix/WindowImplX11.cpp
index da69750525..5f538f46d6 100644
--- a/src/SFML/Window/Unix/WindowImplX11.cpp
+++ b/src/SFML/Window/Unix/WindowImplX11.cpp
@@ -1556,7 +1556,7 @@ void WindowImplX11::initialize()
                                    m_window,
                                    XNInputStyle,
                                    XIMPreeditNothing | XIMStatusNothing,
-                                   reinterpret_cast<void*>(NULL));
+                                   NULL);
     }
     else
     {
